nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -123,6 +124,12 @@ nav_msgs::msg::Path NavfnPlanner::createPlan(
   steady_clock::time_point a = steady_clock::now();
 #endif
 
+  // Refresh parameter
```

Comment:
This isn't a great way to handle this - instead we can use the parameter events callbacks to change variables so its event based and not just polling (see DWB which does this already)